Web/Edubase.Web.UI/Web.config

```diff
@@ -165,8 +165,8 @@
     <add key="api:Username" value="rest-api-user" />
     <add key="api:Password" value="" />
 
-    <add key="LookupApiBaseAddress" value="https://s158d01-func-dd-restapi.azurewebsites.net/api/" />
-    <add key="LookupApiUsername" value="" />
+    <add key="LookupApiBaseAddress" value="https://app-api-t1dv-edubase.azurewebsites.net/edubase/rest/" />
```

Comment:
has there just been another PR that fully removed the value, so it requires explicit enty?

Reply:
I did the web.config changes to remove use of the c# API as a PR to go straight into dev and then I'll merge that into epic/16-19 branch so that line in the web.config that specifies the LookupApiBaseAddress will be removed.